src/eos-metrics-instrumentation.c

```diff
@@ -290,18 +264,26 @@ quit_main_loop (GMainLoop *main_loop)
 }
 
 gint
-main (gint                argc,
-      const gchar * const argv[])
+main (gint  argc,
+      char *argv[])
 {
+  g_autoptr(GOptionContext) context = NULL;
+  g_autoptr(GError) error = NULL;
+
+  context = g_option_context_new ("- record metrics for systemwide events");
+  if (!g_option_context_parse (context, &argc, &argv, &error))
+    {
+      g_print ("option parsing failed: %s\n", error->message);
+      exit (1);
+      }
```

Comment:
Closing brace indentation and I think `g_printerr` rather than `g_print`.

Reply:
I copy-pasted this from the GLib documentation :-)